sycl/plugins/unified_runtime/ur/adapters/native_cpu/program.cpp

```diff
@@ -24,6 +24,11 @@ urProgramCreateWithIL(ur_context_handle_t hContext, const void *pIL,
   DIE_NO_IMPLEMENTATION
 }
 
+struct nativecpu_entry {
+  const char *kernelname;
+  const unsigned char *kernel_ptr;
+};
+
```

Comment:
shouldn't this definition come from some header file?

Reply:
Yeah thanks for pointing it out, I moved the definition to `program.hpp`